pkg/minikube/sysinit/systemd.go

```diff
@@ -63,7 +63,7 @@ func (s *Systemd) Enable(svc string) error {
 
 // Start starts a service
 func (s *Systemd) Start(svc string) error {
-	if err := s.reload(); err != nil {
+	if err := s.daemonReload(); err != nil {
```

Comment:
@medyagh do you remember why this was added? daemon-reload is not needed for starting, reloading, or restarting a service.